documentation/slate/source/includes/isochrones.md

```diff
@@ -11,19 +11,27 @@ the quickest way to reach your destination.
 Isochrone computing exposes information under two formats:
 
 -   either [Journeys](#journeys) service which provides a list with all the reachable stops from a potential destination 
-with their respective arrival times, travel times and number of matches
+with their respective arrival times, travel times and number of matches, here is a fiddle example:
+
```

Comment:
with their respective arrival times, travel times and number of matches**.** Here is a fiddle example: